docs/source/1.0/spec/aws/aws-json-1_0-protocol.rst

```diff
@@ -61,4 +61,208 @@ See
     This protocol only permits the :ref:`httpPayload-trait` to be applied to
     members that target structures, documents, strings, blobs, or unions.
 
-*TODO: Add specifications, protocol examples, etc.*
+
+----------------
+Supported traits
+----------------
+
+The ``awsJson1_0`` protocol supports the following traits
+that affect serialization:
+
+.. list-table::
+    :header-rows: 1
+    :widths: 20 80
+
+    * - Trait
+      - Description
+    * - :ref:`jsonName <jsonName-trait>`
+          - By default, the JSON property names used in serialized structures are
+            the same as a structure member name. The ``jsonName`` trait changes
+            the JSON property name to a custom value.
+
+
+----------------
+Protocol Headers
```

Comment:
What about merging this section and `HTTP Bindings` in to one `Protocol Behavior` section? I think leading with the content down in `HTTP Bindings` is the most important, then followed by these header notes.